### PR TITLE
register/login/logout/destroy

### PR DESCRIPTION
**Pull request by AbdullohHusanov**

### CODE REVIEW

**Comment by AbdullohHusanov** [comment]
dsfsg

File: app/Http/Controllers/RequestController.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ public function show(Request $request)
     /**
      * Update the specified resource in storage.
      */
-    public function update(Request $request)
+    public function update(UpdateRequestRequest $request)
     {
         //
```
Comment (AbdullohHusanov) [inline app/Http/Controllers/RequestController.php]: delete

**Review by AbdullohHusanov — COMMENTED** [review]
good to go

**Review by Abdulhakim73 — COMMENTED** [review]
good to go

**Comment by Abdulhakim73** [comment]
good

**Review by AbdullohHusanov — COMMENTED** [review]
ok

**Pull request closed without merging**